util/parallel.cpp

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) 2018-present The Alive2 Authors.
+// Distributed under the MIT license that can be found in the LICENSE file.
+
+#include "llvm/Support/ErrorHandling.h"
```

Comment:
this part of the code can't depend on LLVM

Reply:
ugh thanks yes I was just figuring this out on my own :(